q2_gatk/plugin_setup.py

```diff
@@ -164,3 +171,23 @@
     "like an index on a database. Note that this tool cannot be run on SAM files, and that the input BAM file must be sorted"
     " in coordinate order.",
 )
+
+plugin.methods.register_function(
+    function=q2_gatk.create_sequence_dictionary,
+    inputs={
+        "reference_sequences": FeatureData[Sequence | GenBankSequence],
```

Comment:
The reference_sequences input should already be in the index_sequences object, so it's not needed here

Reply:
Great catch. I'll remove that input.